docs/src/pages/getting-started/page-layout-examples/page-layout-examples.md

```diff
@@ -11,19 +11,11 @@ for reuse in other pages.
 For multi-part examples, a table in the README at the linked source code location describes
 the purpose of each file.
 
+> For complete ready-to-go examples with multiple page layouts, check out the [premium themes](https://themes.material-ui.com) section.
+
 {{"demo": "pages/getting-started/page-layout-examples/PageLayoutExamples.js", "hideHeader": true}}
 
 If while using these examples you make changes or enhancements that could improve the
 developer experience, or you would like to contribute an additional example,
 please consider creating a [pull request on GitHub](https://github.com/mui-org/material-ui/pulls).
 
-So far we have demos for:
-
-- A dashboard
```

Comment:
What's the motivation for removing those lines? I have added them in order to solve no matches on popular Algolia search queries. For instance, after these changes, the query "dashbard" on Aloglia would yield 0 results.

Reply:
Ah, okay. It seemed redundant to list the examples right after the displaying the examples. Makes sense now.